sdlib/d/gc/extent.d

```diff
@@ -116,6 +122,9 @@ private:
 
 		// Metadata for large extents.
 		size_t usedCapacity;
+
+		// Optional finalizer.
+		void* finalizer;
```

Comment:
Why is this in a union with usedCapacity?

Reply:
Note, the fact that this doesn't break any test is rather alarming. Especially since you zero out finalizer after usedCapacity, which should destroy the used capacity, no?